tokio-trace/tokio-trace-core/src/span.rs

```diff
@@ -19,4 +19,9 @@ impl Span {
     pub fn from_u64(u: u64) -> Self {
         Span(u)
     }
+
+    /// Return the inner `u64` from a span ID.
```

Comment:
TIOLI: Maybe rephrase this as 
```suggestion
    /// Returns the span's ID as a  `u64`.
```
or something?